packages/block-editor/src/components/block-alignment-control/test/index.native.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import BlockAlignmentUI from '../ui';
+
+it( 'should call onChange with undefined, when the control is already active', () => {
+	const onChangeMock = jest.fn();
+	const screen = render(
+		<BlockAlignmentUI value="left" onChange={ onChangeMock } />
+	);
+	const alignButton = screen.getByA11yLabel( 'Align' );
+	fireEvent.press( alignButton );
+	const noneAlignmentButton = screen.getByA11yLabel( 'None' );
+	fireEvent.press( noneAlignmentButton );
+
+	expect( onChangeMock ).toHaveBeenCalledTimes( 1 );
+	expect( onChangeMock ).toHaveBeenCalledWith( undefined );
+} );
+
+it( 'should call onChange with alignment value when the control is inactive', () => {
+	const onChangeMock = jest.fn();
+	const screen = render(
+		<BlockAlignmentUI value="left" onChange={ onChangeMock } />
+	);
+	const alignButton = screen.getByA11yLabel( 'Align' );
+	fireEvent.press( alignButton );
+	const noneAlignmentButton = screen.getByA11yLabel( 'Align center' );
+	fireEvent.press( noneAlignmentButton );
```

Comment:
Since we're getting here the align center button, what do you think about renaming this to `centerAlignmentButton`?

Reply:
Definitely. Thank you for catching this mistake. Addressed in https://github.com/WordPress/gutenberg/pull/35191/commits/bcd16d620af229b66d2f3dd3b22f293552005144.